executor/new_distsql.go

```diff
@@ -123,6 +124,9 @@ func (e *TableReaderExecutor) Next() (Row, error) {
 
 // Open implements the Executor Open interface.
 func (e *TableReaderExecutor) Open() error {
+	span, ctx := startSpanFollowsContext(e.ctx.GoCtx(), "executor.TableReader")
```

Comment:
Why not use `executor.TableReader.Open`?

Reply:
If use `executor.TableReader`, we need to finish the span in `Close`.

Reply:
I just want the information of the parent span of coprocessor rpc:

```
runStmt -> coprocessor rpc    // lost too much information
runStmt -> executor.XXX -> coprocessor rpc  // much better
```

It doesn't matter just log the `Open`, I use `FollowsFrom` rather than `ChildOf` here.